defaults/main.yml

```diff
@@ -24,7 +25,7 @@ yum_cron_daily_update_message: yes
 yum_cron_daily_download_updates: no
 yum_cron_daily_apply_updates: no
 yum_cron_daily_random_sleep: 60
-#yum_cron_daily_base_options: "{{ hourly_base_options }}"
+yum_cron_daily_base_options: "{{ yum_cron_hourly_base_options }}"
```

Comment:
This is definitely a var rename bug that needs fixing, but can you keep it commented out for now like it was previous?

Reply:
Addressed in #15